src/OfflineImage.js

```diff
@@ -37,6 +37,12 @@ class OfflineImage extends React.Component {
     const reloadImage = nextProps.reloadImage;
 
     const source = this.props.source;
+    // new source is from local storage, no need to cache
+    if (nextSource && nextSource.uri.includes('file:///')) {
```

Comment:
Thank you for PR. Sorry for late response. Somehow I did not get a notification. I will enable them now.

* Is this tested both in Android and iOS.
* file:// is this not sufficient and I am not sure how it works for both platform :)
* In the code, we already explicitly add `uri: FILE_PREFIX + this.state.path,` at line 85 in the component. I am little doubt about this as well.

Overall, I am not sure whether we want to support local storage image URI as we can also directly use RN Image instead based on your functionality :)

Thanks!